src/app/views/App.tsx

```diff
@@ -345,7 +345,7 @@ class App extends Component<IAppProps, IAppState> {
     if (mobileScreen) {
       sidebarWidth = layout = 'col-xs-12 col-sm-12';
     }
-    if (permissionModeType !== PERMISSION_MODE_TYPE.TeamsApp && this.state.hideDialog) {
+    if (permissionModeType !== PERMISSION_MODE_TYPE.TeamsApp && hideDialog) {
```

Comment:
hmm if it's `hideDialog === true` it shows dialog?

Reply:
maybe i am misunderstanding, but if it's user mode and hideDialog is true we show dialog?

Reply:
When you exist the app mode (ie go into user mode) and the pop up has been hidden, it's supposed to reset the popup for the next time they switch back to app mode. This is supposed to be a temporary thing and is awaiting the next pr :)